packages/rest/src/rest.server.ts

```diff
@@ -310,6 +310,11 @@ export class RestServer extends Context implements Server, HttpServerLike {
         this._setupOperation(verb, path, routeSpec);
       }
     }
+
+    this.staticAssets.forEach(assetEntry => {
+      const {path, rootDir, options} = assetEntry;
+      this.httpHandler.registerStaticAssets(path, rootDir, options);
```

Comment:
I think you should call `this._httpHandler.registerStaticAssets` here, instead of `this.httpHandler....`?